src/neo/Ledger/SendersFeeMonitor.cs

```diff
@@ -23,14 +23,16 @@ public BigInteger GetSenderFee(UInt160 sender)
             return value;
         }
 
-        public void AddSenderFee(Transaction tx)
+        public BigInteger AddSenderFee(Transaction tx)
         {
+            BigInteger ret = tx.SystemFee + tx.NetworkFee;
```

Comment:
This is just an optimization, right?

I am going to still check the first part of changes on the Blockchain.

Reply:
If you prefer, I can do that with `GetSenderFee` without change this, maybe it's better.

Reply:
For me it is good. It looks a good optimization to compute the value before the lock, right?